src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/UsbPolicy.cs

```diff
@@ -0,0 +1,146 @@
+        private EventHandler<PolicyChangedEventArgs> _usbTetheringPolicyChanged;
+
+        internal UsbPolicy()
+        {
+        }
+
+        internal UsbPolicy(DevicePolicyManager dpm) : base(dpm)
+        {
+        }
+
+        /// <summary>
+        /// A Destructor of UsbPolicy.
+        /// </summary>
+        ~UsbPolicy()
+        {
+            this.Dispose(false);
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object.
+        /// </summary>
+        public void Dispose()
+        {
+            this.Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
+        /// </summary>
+        /// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!_disposed)
+            {
+                if (disposing)
+                {
+                    // to be used if there are any other disposable objects
+                }
+
+                if (_usbTetheringCallbackId != 0)
+                {
+                    try
+                    {
+                        RemoveUsbTetheringPolicyChangedCallback();
+                    }
+                    catch (Exception e)
+                    {
+                        Log.Error(Globals.LogTag, e.ToString());
+                    }
+                }
+
+                _disposed = true;
+            }
+        }
+
+        /// <summary>
+        /// The UsbTetheringPolicyChanged event is raised when the usb tethering policy is changed.
+        /// </summary>
+        /// <remarks>This event will be removed automatically when UsbPolicy is destroyed.</remarks>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<PolicyChangedEventArgs> UsbTetheringPolicyChanged
+        {
+            add
+            {
+                if (_usbTetheringPolicyChanged == null)
+                {
+                    AddUsbTetheringPolicyChangedCallback();
+                }
+
+                _usbTetheringPolicyChanged += value;
+            }
+
+            remove
+            {
+                _usbTetheringPolicyChanged -= value;
+                if (_usbTetheringPolicyChanged == null)
+                {
+                    RemoveUsbTetheringPolicyChangedCallback();
+                }
+            }
+        }
+
+        private void AddUsbTetheringPolicyChangedCallback()
+        {
+            if (_usbTetheringPolicyChangedCallback == null)
+            {
+                _usbTetheringPolicyChangedCallback = (string name, string state, IntPtr userData) =>
+                {
+                    _usbTetheringPolicyChanged?.Invoke(null, new PolicyChangedEventArgs(name, state));
```

Comment:
Why pass `null` instead of `this`? `sender` can be null when event is a static
```suggestion
                    _usbTetheringPolicyChanged?.Invoke(this, new PolicyChangedEventArgs(name, state));
```

Reply:
I changed null to this.